lib/ansible/modules/remote_management/hpilo/hponcfg.py

```diff
@@ -40,6 +40,20 @@
   minfw:
     description:
     - The minimum firmware level needed
+    required: false
+  hponcfg:
+    description:
+    - Path to the hponcfg executable (`hponcfg` which uses $PATH)
+    default: hponcfg
+    required: false
```

Comment:
This is not wanted, a parameter is considered optional by default unless you add `required: yes`.